lib/mysql.js

```diff
@@ -650,15 +669,23 @@ MySQL.prototype.count = function count(model, callback, where) {
  * @param {Function} [callback] The callback function
  *
  */
-MySQL.prototype.destroyAll = function destroyAll(model, where, callback) {
+MySQL.prototype.deleteAll =
+MySQL.prototype.destroyAll = function(model, where, callback) {
   if (!callback && 'function' === typeof where) {
     callback = where;
     where = undefined;
   }
   this.query('DELETE FROM '
       + this.tableEscaped(model) + ' ' + this.buildWhere(model, where || {}),
-    function (err, data) {
-      callback && callback(err, data);
+    function(err, info) {
+      // @superkhau The second arg is purposely set to an empty array for
+      // consistency across all connectors. The postgres connector currently
+      // returns `data.rows` (which is an array) in the second arg and cannot be
+      // changed for backwards compatibility reasons. So instead, we leave the
+      // second arg alone and pass metadata in the third arg instead.
+      callback && callback(err, [], {
```

Comment:
We need to preserve backwards compatibility of the mysql connector too! IMO you should keep passing `data` as the second callback argument.

I.e. any user code calling juggler and mysql-connector should keep receiving `data` in the callback.